### PR TITLE
🐛 Fix: Flexible Login Page

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

After a resize, when reloading the different login pages in the stack, we used to recreate them loosing context (which page was selected) in the stack. This PR fixes it by not recreating the views and the stack after switching to the compact view:

FlexLogin

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by jsaq007 — COMMENTED** [review]
👌

File: services/static-webserver/client/source/class/osparc/auth/LoginPage.js [hunk 1]
```diff
@@ -40,6 +40,91 @@ qx.Class.define("osparc.auth.LoginPage", {
   },
 
   members: {
+    _createChildControlImpl: function(id) {
```
Comment (ignapas) [inline services/static-webserver/client/source/class/osparc/auth/LoginPage.js:43]: 😵 👍🏼

**Review by ignapas — APPROVED** [review]
nice

File: services/static-webserver/client/source/class/osparc/auth/LoginPage.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/auth/LoginPage.js:43]: :D

**Review by GitHK — COMMENTED** [review]
👍

**Review by pcrespov — COMMENTED** [review]
👍

**Pull request merged**